### ISSUE TITLE
Installing from any channel other than conda-forge and default results in 404 error

### ISSUE THREAD

**Issue by nup002** [conda/conda#13158]
### What happened?

I am unable to install packages from any other channel than `conda-forge` and `default`. Attempting to install from other channels will invariably result in a UnavailableInvalidChannel 404 error. 

**Repro steps:**

1. Install miniconda
2. Create a new conda environment
3. Update conda with `conda update --all`
4. Try to install a package _not_ from `conda-forge` or `default`

Example of command that fails, and the output it produces:
```
> conda install pyyaml -c intel

UnavailableInvalidChannel: HTTP 404 NOT FOUND for channel intel <https://conda.anaconda.org/intel>

The channel is not accessible or is invalid.

You will need to adjust your conda configuration to proceed.
Use `conda config --show channels` to view your configuration's current state,
and use `conda config --show-sources` to view config file locations.
```

Giving `conda-forge` or `default` as the channel arguments works just fine. For example, the following command executes without problems: `conda install numpy -c conda-forge`.

At the bottom of this post I have added the verbose output from the above command. There, on line 3, it appears to receive its first 404 response when trying to access ```https://conda.anaconda.org/intel/win-64/current_repodata.json```. Plugging this address into a web browser does indeed give a 404 response. This may be a smoking gun.

**Steps I have tried to resolve my problems:**

- Full uninstall of miniconda with deletion of all related files (`.condarc`, `Users/miniconda3`, etc)
- Restart of PC
- Different networks (wifi, cable, mobile data via hotspot)
- VPNs to see if it's an issue with my location (it's not)
- Turn off PC firewall
- Adding the channels to `condarc`

I have also tested it on a different PC, which does not have this problem. This makes me suspicious that something is broken in the particular PC I am experiencing this problem on.

Verbose output from `conda install pyyaml -c intel`:
```
C:\Users\magla\Documents\Repositories\widc.app>conda install pyyaml -c intel -v
Collecting package metadata (current_repodata.json): ...working... INFO conda.gateways.repodata:conda_http_errors(230): Unable to retrieve repodata (response: 404) for https://conda.anaconda.org/intel/win-64/current_repodata.json
INFO conda.gateways.repodata:conda_http_errors(230): Unable to retrieve repodata (response: 404) for https://conda.anaconda.org/intel/win-64/repodata.json
failed
Traceback (most recent call last):
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 188, in conda_http_errors
    yield
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 154, in repodata
    response.raise_for_status()
  File "C:\Users\magla\miniconda3\lib\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://conda.anaconda.org/t/.../intel/noarch/current_repodata.json

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\subdir_data.py", line 322, in _load
    repodata, state = fetcher.fetch_latest_parsed()
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 758, in fetch_latest_parsed
    parsed, state = self.fetch_latest()
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 880, in fetch_latest
    raw_repodata = repo.repodata(cache.state)  # type: ignore
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 144, in repodata
    with conda_http_errors(self._url, filename):
  File "C:\Users\magla\miniconda3\lib\contextlib.py", line 153, in __exit__
    self.gen.throw(typ, value, traceback)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 253, in conda_http_errors
    raise UnavailableInvalidChannel(
conda.exceptions.UnavailableInvalidChannel: HTTP 404 NOT FOUND for channel intel <https://conda.anaconda.org/intel>

The channel is not accessible or is invalid.

You will need to adjust your conda configuration to proceed.
Use `conda config --show channels` to view your configuration's current state,
and use `conda config --show-sources` to view config file locations.


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 188, in conda_http_errors
    yield
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 154, in repodata
    response.raise_for_status()
  File "C:\Users\magla\miniconda3\lib\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://conda.anaconda.org/t/.../intel/noarch/repodata.json

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\exception_handler.py", line 17, in __call__
    return func(*args, **kwargs)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\cli\main.py", line 64, in main_subshell
    exit_code = do_call(args, parser)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\cli\conda_argparse.py", line 167, in do_call
    result = getattr(module, func_name)(args, parser)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\notices\core.py", line 124, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\cli\main_install.py", line 26, in execute
    install(args, parser, "install")
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\cli\install.py", line 316, in install
    unlink_link_transaction = solver.solve_for_transaction(
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\solve.py", line 154, in solve_for_transaction
    unlink_precs, link_precs = self.solve_for_diff(
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\solve.py", line 215, in solve_for_diff
    final_precs = self.solve_final_state(
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\solve.py", line 358, in solve_final_state
    ssc = self._collect_all_metadata(ssc)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\common\io.py", line 84, in decorated
    return f(*args, **kwds)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\solve.py", line 572, in _collect_all_metadata
    index, r = self._prepare(prepared_specs)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\solve.py", line 1286, in _prepare
    reduced_index = get_reduced_index(
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\index.py", line 287, in get_reduced_index
    new_records = SubdirData.query_all(
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\subdir_data.py", line 158, in query_all
    result = tuple(
  File "C:\Users\magla\miniconda3\lib\concurrent\futures\_base.py", line 621, in result_iterator
    yield _result_or_cancel(fs.pop())
  File "C:\Users\magla\miniconda3\lib\concurrent\futures\_base.py", line 319, in _result_or_cancel
    return fut.result(timeout)
  File "C:\Users\magla\miniconda3\lib\concurrent\futures\_base.py", line 451, in result
    return self.__get_result()
  File "C:\Users\magla\miniconda3\lib\concurrent\futures\_base.py", line 403, in __get_result
    raise self._exception
  File "C:\Users\magla\miniconda3\lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\subdir_data.py", line 143, in subdir_query
    return tuple(
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\subdir_data.py", line 165, in query
    self.load()
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\subdir_data.py", line 265, in load
    _internal_state = self._load()
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\subdir_data.py", line 327, in _load
    return self._load()
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\core\subdir_data.py", line 322, in _load
    repodata, state = fetcher.fetch_latest_parsed()
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 758, in fetch_latest_parsed
    parsed, state = self.fetch_latest()
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 880, in fetch_latest
    raw_repodata = repo.repodata(cache.state)  # type: ignore
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 144, in repodata
    with conda_http_errors(self._url, filename):
  File "C:\Users\magla\miniconda3\lib\contextlib.py", line 153, in __exit__
    self.gen.throw(typ, value, traceback)
  File "C:\Users\magla\miniconda3\lib\site-packages\conda\gateways\repodata\__init__.py", line 253, in conda_http_errors
    raise UnavailableInvalidChannel(
conda.exceptions.UnavailableInvalidChannel: HTTP 404 NOT FOUND for channel intel <https://conda.anaconda.org/intel>

The channel is not accessible or is invalid.

You will need to adjust your conda configuration to proceed.
Use `conda config --show channels` to view your configuration's current state,
and use `conda config --show-sources` to view config file locations.
```

### Conda Details

<details>
<summary><code>conda info</code></summary>

```shell
active environment : intel_install_test
    active env location : C:\Users\magla\miniconda3\envs\intel_install_test
            shell level : 2
       user config file : C:\Users\magla\.condarc
 populated config files : C:\Users\magla\.condarc
          conda version : 23.7.4
    conda-build version : not installed
         python version : 3.10.12.final.0
       virtual packages : __archspec=1=x86_64
                          __cuda=12.2=0
                          __win=0=0
       base environment : C:\Users\magla\miniconda3  (writable)
      conda av data dir : C:\Users\magla\miniconda3\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\magla\miniconda3\pkgs
                          C:\Users\magla\.conda\pkgs
                          C:\Users\magla\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\magla\miniconda3\envs
                          C:\Users\magla\.conda\envs
                          C:\Users\magla\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/23.7.4 requests/2.31.0 CPython/3.10.12 Windows/10 Windows/10.0.22000
          administrator : False
             netrc file : None
           offline mode : False
```

</details>

<details>
<summary><code>conda config --show-sources</code></summary>

```shell
==> C:\Users\magla\.condarc <==
ssl_verify: True
channels:
  - conda-forge
  - defaults

==> cmd_line <==
debug: False
json: False
```

</details>

<details>
<summary><code>conda config --show</code></summary>

```shell
add_anaconda_token: True
add_pip_as_python_dependency: True
aggressive_update_packages:
  - ca-certificates
  - certifi
  - openssl
allow_conda_downgrades: False
allow_cycles: True
allow_non_channel_urls: False
allow_softlinks: False
allowlist_channels: []
always_copy: False
always_softlink: False
always_yes: None
anaconda_upload: None
auto_activate_base: True
auto_stack: 0
auto_update_conda: True
bld_path:
changeps1: True
channel_alias: https://conda.anaconda.org
channel_priority: flexible
channel_settings: []
channels:
  - conda-forge
  - defaults
client_ssl_cert: None
client_ssl_cert_key: None
clobber: False
conda_build: {}
create_default_packages: []
croot: C:\Users\magla\miniconda3\conda-bld
custom_channels:
  pkgs/main: https://repo.anaconda.com
  pkgs/r: https://repo.anaconda.com
  pkgs/msys2: https://repo.anaconda.com
  pkgs/pro: https://repo.anaconda.com
custom_multichannels:
  defaults:
    - https://repo.anaconda.com/pkgs/main
    - https://repo.anaconda.com/pkgs/r
    - https://repo.anaconda.com/pkgs/msys2
  local:
debug: False
default_channels:
  - https://repo.anaconda.com/pkgs/main
  - https://repo.anaconda.com/pkgs/r
  - https://repo.anaconda.com/pkgs/msys2
default_python: 3.10
default_threads: None
deps_modifier: not_set
dev: False
disallowed_packages: []
download_only: False
dry_run: False
enable_private_envs: False
env_prompt: ({default_env})
envs_dirs:
  - C:\Users\magla\miniconda3\envs
  - C:\Users\magla\.conda\envs
  - C:\Users\magla\AppData\Local\conda\conda\envs
error_upload_url: https://conda.io/conda-post/unexpected-error
execute_threads: 1
experimental: []
extra_safety_checks: False
fetch_threads: 5
force: False
force_32bit: False
force_reinstall: False
force_remove: False
ignore_pinned: False
json: False
local_repodata_ttl: 1
migrated_channel_aliases: []
migrated_custom_channels: {}
no_plugins: False
non_admin_enabled: True
notify_outdated_conda: True
number_channel_notices: 5
offline: False
override_channels_enabled: True
path_conflict: clobber
pinned_packages: []
pip_interop_enabled: False
pkgs_dirs:
  - C:\Users\magla\miniconda3\pkgs
  - C:\Users\magla\.conda\pkgs
  - C:\Users\magla\AppData\Local\conda\conda\pkgs
proxy_servers: {}
quiet: False
remote_backoff_factor: 1
remote_connect_timeout_secs: 9.15
remote_max_retries: 3
remote_read_timeout_secs: 60.0
repodata_fns:
  - current_repodata.json
  - repodata.json
repodata_threads: None
report_errors: None
restore_free_channel: False
rollback_enabled: True
root_prefix: C:\Users\magla\miniconda3
safety_checks: warn
sat_solver: pycosat
separate_format_cache: False
shortcuts: True
show_channel_urls: None
signing_metadata_url_base: None
solver: classic
solver_ignore_timestamps: False
ssl_verify: True
subdir: win-64
subdirs:
  - win-64
  - noarch
target_prefix_override:
track_features: []
unsatisfiable_hints: True
unsatisfiable_hints_check_depth: 2
update_modifier: update_specs
use_index_cache: False
use_local: False
use_only_tar_bz2: False
verbosity: 0
verify_threads: 1
```

</details>

<details>
<summary><code>conda list</code></summary>

```shell
# packages in environment at C:\Users\magla\miniconda3\envs\intel_install_test:
#
# Name                    Version                   Build  Channel
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2023.7.22            h56e8100_0    conda-forge
libexpat                  2.5.0                h63175ca_1    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libsqlite                 3.43.0               hcfcfb64_0    conda-forge
libzlib                   1.2.13               hcfcfb64_5    conda-forge
logbook                   1.6.0           py311ha68e1ae_1    conda-forge
openssl                   3.1.3                hcfcfb64_0    conda-forge
pip                       23.2.1             pyhd8ed1ab_0    conda-forge
python                    3.11.5          h2628c8c_0_cpython    conda-forge
python_abi                3.11                    4_cp311    conda-forge
pyzmq                     25.1.1          py311h7b3f143_1    conda-forge
setuptools                68.2.2             pyhd8ed1ab_0    conda-forge
termcolor                 2.3.0              pyhd8ed1ab_0    conda-forge
tk                        8.6.13               hcfcfb64_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
vc                        14.3                h64f974e_17    conda-forge
vc14_runtime              14.36.32532         hdcecf7f_17    conda-forge
vs2015_runtime            14.36.32532         h05e6639_17    conda-forge
wheel                     0.41.2             pyhd8ed1ab_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
zeromq                    4.3.4                h0e60522_1    conda-forge
```

</details>

**Comment by kenodegard** [comment]
I am unable to replicate this issue

It looks like you have a token enabled, that may perhaps be messing with the channels, try logging out (`anaconda logout`) and see if that changes anything

**Comment by jezdez** [comment]
If this issue persists, please file a ticket in the [Anaconda issue tracker](https://github.com/ContinuumIO/anaconda-issues/issues) as this probably is related to repo hosting and anaconda.org, not the conda package manager software that this issue tracker is about. Thank you for understanding!

**Comment by travishathaway** [comment]
Closing as https://github.com/conda/conda/labels/off-topic

**Comment by nup002** [comment]
> I am unable to replicate this issue
> 
> It looks like you have a token enabled, that may perhaps be messing with the channels, try logging out (`anaconda logout`) and see if that changes anything

Thank you, installing Anaconda and running `anaconda logout` fixed the problem. I was at first skeptical that this would help since I did not even have Anaconda installed, only Miniconda (which does not have a `logout` command), but I gave it a try anyways.

What might have happened is that I used to have Anaconda installed. I then logged in, and later uninstalled Anaconda in favor of Miniconda. Somehow my login state was kept around even though Anaconda was gone. I hope this thread help someone in the future with the same problem!
